### PR TITLE
v0.0.55 - Fixed bugs in Model3DViewer

### PR DESCRIPTION
**Pull request by sharonsun1999**
Modified CSS properties of HTML component so that sizes of buttons and sliders can change as the size of parent component changes;
Fixed the bug that axis labels of sliders all display 'x';
Removed slicing planes from states and made them into normal variables so that the component does not need to be re-rendered every time users drag the sliders;

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-417.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
@dlcognite Could you review this and make sure it doesn't mess up your v1 migration?

File: src/components/Model3DViewer/Model3DViewer.tsx [hunk 1]
```diff
@@ -56,26 +56,19 @@ export interface Model3DViewerProps {
...
-interface Model3DViewerState {
-  planes: THREE.Plane[];
-  flipped: [boolean, boolean, boolean];
-}
-
 export function mockCreateViewer(mockFunction: any) {
   createViewer = mockFunction || originalCreateViewer;
 }
 
 const containerStyles = {
```
Comment (dlcognite) [inline src/components/Model3DViewer/Model3DViewer.tsx]: Is it possible to use styled component here instead?

File: src/components/Model3DViewer/Model3DViewer.tsx [hunk 2]
```diff
@@ -383,9 +374,9 @@ export class Model3DViewer extends React.Component<
         style={{
```
Comment (dlcognite) [inline src/components/Model3DViewer/Model3DViewer.tsx]: Can we also convert this to styled component?

File: src/components/Model3DViewer/Model3DViewer.tsx [hunk 3]
```diff
@@ -399,46 +390,42 @@ export class Model3DViewer extends React.Component<
...
-      <div style={{ width: '100vw' }}>
+      <div
+        style={{
+          width: '100%',
+          height: '100%',
+          position: 'relative',
+        }}
+      >
         {this.props.showScreenshotButton ? (
           <div
```
Comment (dlcognite) [inline src/components/Model3DViewer/Model3DViewer.tsx]: styled component :D

**Pull request merged**